single-node/src/Explicit-Lagrange-Kokkos/SGH_solver/force_sgh.cpp

```diff
@@ -2,275 +2,261 @@
 #include "mesh.h"
 #include "state.h"
 
-
-
-
-
 // -----------------------------------------------------------------------------
 // This function calculates the corner forces and the evolves stress (hypo)
-//------------------------------------------------------------------------------
-void get_force_sgh(const CArrayKokkos <material_t> &material,
-                   const mesh_t &mesh,
-                   const DViewCArrayKokkos <double> &node_coords,
-                   const DViewCArrayKokkos <double> &node_vel,
-                   const DViewCArrayKokkos <double> &elem_den,
-                   const DViewCArrayKokkos <double> &elem_sie,
-                   const DViewCArrayKokkos <double> &elem_pres,
-                   const DViewCArrayKokkos <double> &elem_stress,
-                   const DViewCArrayKokkos <double> &elem_sspd,
-                   const DViewCArrayKokkos <double> &elem_vol,
-                   const DViewCArrayKokkos <double> &elem_div,
-                   const DViewCArrayKokkos <size_t> &elem_mat_id,
-                   DViewCArrayKokkos <double> &corner_force,
-                   const double fuzz,
-                   const double small,
-                   const DViewCArrayKokkos <double> &elem_statev,
-                   const double dt,
-                   const double rk_alpha
-                   ){
-    
-
-    
+// ------------------------------------------------------------------------------
+void
+get_force_sgh(const CArrayKokkos<material_t>&  material,
+              const mesh_t&                    mesh,
+              const DViewCArrayKokkos<double>& node_coords,
+              const DViewCArrayKokkos<double>& node_vel,
+              const DViewCArrayKokkos<double>& elem_den,
+              const DViewCArrayKokkos<double>& elem_sie,
+              const DViewCArrayKokkos<double>& elem_pres,
+              const DViewCArrayKokkos<double>& elem_stress,
+              const DViewCArrayKokkos<double>& elem_sspd,
+              const DViewCArrayKokkos<double>& elem_vol,
+              const DViewCArrayKokkos<double>& elem_div,
+              const DViewCArrayKokkos<size_t>& elem_mat_id,
+              DViewCArrayKokkos<double>&       corner_force,
+              const double                     fuzz,
```

Comment:
Also not sure these large indents to force alignment are good. Same goes for the repeated assignments in functions where this spacing was taking place. I think it being on its own row is enough to draw attention.

Reply:
My only personal argument for this is that I get warm fuzzies whenever things are aligned.  Also, the Google C++ style guide suggests all parameters should be aligned if possible, but it is unclear if they mean the types or both types and parameter names. I am happy to switch back if others prefer the original. @nathanielmorgan @djdunning  Thoughts?

Reply:
I like alignment of arguments!!!

Reply:
I specifically mean the large spaces between the type names and the variable names, left alignment is a must.

Reply:
I'm fine with the gap between the type and the variable name, it might make it easier to see the names

Reply:
thanks Adrian for the clarification

Reply:
I think it'll just look weird in the extreme case where one type name is very long and then every other argument is halfway across the screen.

Reply:
Ok

Reply:
Yay democracy!  Ill leave this open for a bit to see if anyone else has thoughts.